### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by AdithyaBaglody**
1. This PR removes the old reference for EPID.
2. This PR updates the docs to reflect the new build system.

### CODE REVIEW

File: docs/build_conf.md [hunk 1]
```diff
@@ -19,23 +19,80 @@ There following are some of the options to choose when building the device:
...
+DA_FILE=pem           # only Use if ECDSA private keys are PEM encoded
+
+List of Public Key encoding/owner-attestation options:
+PK_ENC=rsa            # Use RSAMODEXP-RSA2048RESTR public key encoding
+PK_ENC=ecdsa          # Use ECDSA-X.509 based public key encoding (default)
+
+Underlying crypto library to be used:
+TLS=openssl           # (Linux default, not supported for other TARGET_OS)
+TLS=mbedtls           # (Mbed OS default, not supported for other TARGET_OS)
+CRYPTO_HW=true        # Use Secure element for some of the crypto operations
```
Comment (trbehera) [inline docs/build_conf.md:75]: Should we mention that the default value is false?

File: docs/build_conf.md [hunk 2]
```diff
@@ -19,23 +19,80 @@ There following are some of the options to choose when building the device:
...
+
+Option to enable network-proxy:
+HTTPPROXY=true        # http-proxy enabled (default)
+HTTPPROXY=false       # http-proxy disabled
+PROXY_DISCOVERY=true  # network discovery enabled
```
Comment (trbehera) [inline docs/build_conf.md:80]: Same here.

File: docs/secure_element.md [hunk 3]
```diff
@@ -70,7 +70,8 @@ SDO implementation for ARM devices uses Microchip AT608A as the secure element.
    Note: only ECDSA is supported from SE.
    ```bash
    cd <c-code-sdk folder>
-   make CRYPTO_HW=true PK_ENC=ecdsa KEX=ecdh
+   cmake -DCRYPTO_HW=true -DPK_ENC=ecdsa -DKEX=ecdh .
```
Comment (trbehera) [inline docs/secure_element.md:73]: For uniformity, we might start the commands with $. 

Can be taken up at a later time.

File: docs/build_conf.md (hunk 1 above)
Comment (AdithyaBaglody) [reply docs/build_conf.md:75]: this is showing only the possible build configs. doesn't exactly represent the selected values. 
That will be shown at compile time.

Comment (nbmaiti) [reply docs/build_conf.md:75]: Instead  "Use Secure element for some of the crypto operations  ", write true/false allowed, default=false

**Comment by AdithyaBaglody** [comment]
@trbehera @nbmaiti revist. i have update the PR for the review fixes

**Pull request merged**